docs/sdk/v3/guides/03-trading.md

```diff
@@ -59,22 +66,22 @@ This example uses an exact input trade, but we can also construct a trade using
 
 Once we have created a trade, we can now execute this trade with our provider. First, we must set our options that define how much time and slippage can occur in our execution as well as the address to use for our wallet:
 
-```js reference title="Constructing SwapOptions" referenceLinkText="View on Github" customStyling
+```typescript reference title="Constructing SwapOptions" referenceLinkText="View on Github" customStyling
 https://github.com/Uniswap/examples/blob/e8bd4178ccaccd6776407f79a319128d4c31f90d/v3-sdk/trading/src/trading.ts#L86-L90
 ```
 
 Next, we use the Uniswap `SwapRouter` to get the associated call parameters for our trade and options:
 
-```js reference title="Getting call parameters" referenceLinkText="View on Github" customStyling
+```typescript reference title="Getting call parameters" referenceLinkText="View on Github" customStyling
 https://github.com/Uniswap/examples/blob/e8bd4178ccaccd6776407f79a319128d4c31f90d/v3-sdk/trading/src/trading.ts#L92
 ```
 
 Finally, we can construct a transaction from the method parameters and send the transaction:
 
-```js reference title="Sending a transaction" referenceLinkText="View on Github" customStyling
+```typescript reference title="Sending a transaction" referenceLinkText="View on Github" customStyling
 https://github.com/Uniswap/examples/blob/e8bd4178ccaccd6776407f79a319128d4c31f90d/v3-sdk/trading/src/trading.ts#L94-L101
 ```
 
-## Result and Next Steps
+## Next Steps
 
-The resulting example allows for trading between any two ERC20 tokens, but this can be suboptimal for the best pricing and fees. To achieve the best possible price, we use the Uniswap auto router to route through pools to get an optimal cost. The next guide on [routing](./04-routing.md) will show you how to use this router and execute optimal swaps.
\ No newline at end of file
+The resulting example allows for trading between any two ERC20 tokens, but this can be suboptimal for the best pricing and fees. To achieve the best possible price, we use the Uniswap auto router to route through pools to get an optimal cost. Our [routing](./04-routing.md) guide will show you how to use this router and execute optimal swaps.
```

Comment:
We could add the ability to trade directly from ETH if we want to

Reply:
This isn't true in the example actually - we've specifically made it tokens only so we don't have the native currency case